tests/DoctrineTest/Coverage.php

```diff
@@ -138,7 +140,8 @@ public function getRevision()
      */
     public function generateReport()
     {
-        $svn_info = explode(" ", exec("svn info | grep Revision"));
+        // $svn_info = explode(" ", exec("svn info | grep Revision"));
+        $svn_info = 'svn is not used anymore. maybe adding some git infos or drop it?';
```

Comment:
What are your thoughts on this? I tend to drop it completely.

Reply:
Or we should migrate back to svn. 🤔
drop it

Reply:
I've removed this part, trigger a deprecation warning and return 0 in case anyone uses the public method. Otherwise this little change might imply a BC

Reply:
If we do not remove it, the version format change would also be BC.

Reply:
It's removed, I just decided to continue the discussion on the now outdated file to keep it together :)

Reply:
That's fine, but I thought I'd note that if it's not removed it's still BC, so I'd prefer removal.